### ISSUE TITLE
Not working in BackgroundTask

### ISSUE THREAD

**Issue by iurifarenzena** [selomkofori/ds18b20_win10iot#4]
I'm facing some trouble on trying to use this library in a background task. It randomly freezes at some operations like read/write and even dispose of the serialPort.
The sensor is correctly wired, cause the sample application works very well in foreground.
Any idea?

**Comment by epastorejr** [comment]
Yeah, if someone could come up with an idea on background running...
